sim/core/flags.go

```diff
@@ -165,6 +165,7 @@ const (
 	SpellFlagCannotBeDodged                                // Ignores dodge in physical hit rolls
 	SpellFlagBinary                                        // Does not do partial resists and could need a different hit roll.
 	SpellFlagChanneled                                     // Spell is channeled
+	SpellFlagCastWhileChanneling                           // Spell can be cast while channeling
```

Comment:
Also add to this comment: "If SpellFlagChanneled and SpellFlagCastWhileChanneling are both set, it means that *other* spells with the SpellFlagCastWhileChanneling flag can be cast without interrupting the channeled spell."